packages/scenes/src/components/SceneRefreshPicker.tsx

```diff
@@ -11,6 +11,10 @@ import { SceneObjectUrlSyncConfig } from '../services/SceneObjectUrlSyncConfig';
 
 export const DEFAULT_INTERVALS = ['5s', '10s', '30s', '1m', '5m', '15m', '30m', '1h', '2h', '1d'];
 
+function filterDissalowedIntervals(i: string){
```

Comment:
```suggestion
function filterDissalowedIntervals(i: string) {
```

Nit but don't forget to run prettier before merging.